ion/src/main/java/com/fasterxml/jackson/dataformat/ion/IonParser.java

```diff
@@ -40,6 +40,18 @@
      */
     public enum Feature implements FormatFeature // in 2.12
     {
+        /**
+         * Whether to expect Ion native Type Id construct for indicating type (true);
+         * or "generic" type property (false) when deserializing.
+         *<p>
+         * Enabled by default for backwards compatibility as that has been the behavior
+         * of `jackson-dataformat-ion` since 2.9 (first official public version)
+         *
+         * @see <a href="https://amzn.github.io/ion-docs/docs/spec.html#annot">The Ion Specification</a>
+         *
+         * @since 2.12
```

Comment:
If added at this point, must be marked as since 2.12.3 (but I'll add another note on this)

Reply:
Got it. I will update it.